print('this is okay')

raise ValueError('this is an error')
